### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by Gathondu**
#### What does this PR do?
Refactor tests for the scrapers
Refactor tests for the scraper tests

### CODE REVIEW

File: healthtools/config.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 # Where we archive the data
 DATA_DIR = os.getcwd() + "/data/"
-TEST_DIR = DATA_DIR + "tests/"
+TEST_DIR = os.getcwd() + "/healthtools/tests/"
```
Comment (DavidLemayian) [inline healthtools/config.py:27]: Any reason why? The `TEST_DIR` is where the test data would go.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: healthtools/config.py (hunk 1 above)
Comment (Gathondu) [reply healthtools/config.py:27]: the `TEST_DIR` is where the json data files used are located

Comment (DavidLemayian) [reply healthtools/config.py:27]: @Gathondu yeah, we're saying the same thing. So wouldn't it therefore go to the local data folder then? Or actually wherever the `DATA_DIR` is pointed to?

File: healthtools/tests/test_scrapers.py [hunk 2]
```diff
@@ -23,110 +23,110 @@ def setUp(self):
...
         self.assertTrue(len(entries) > 1)
 
     def test_it_scrapes_nhif_outpatient_cs_page(self):
-        entries = self.nhif_outpatient_cs_scraper.scrape_page(1)
+        entries = self.nhif_outpatient_cs_scraper.scrape_page(1, 5)
         self.assertTrue(len(entries) > 1)
 
     def test_doctors_scraper_uploads_to_elasticsearch(self):
-        with open(TEST_DIR + "/dummy_files/doctors.json", "r") as my_file:
+        with open(TEST_DIR + "dummy_files/doctors.json", "r") as my_file:
```
Comment (Gathondu) [inline healthtools/tests/test_scrapers.py:57]: @DavidLemayian this is what I mean. We have a folder in tests that has the dummy files

Comment (Gathondu) [reply healthtools/tests/test_scrapers.py:57]: the dummy files are supposed to act like data pulled from a scraper that the tests uses to test that it uploads

Comment (DavidLemayian) [reply healthtools/tests/test_scrapers.py:57]: @Gathondu And where would test data be downloaded to?

Comment (Gathondu) [reply healthtools/tests/test_scrapers.py:57]: @DavidLemayian to `DATA_DIR` but under a test folder

Comment (Gathondu) [reply healthtools/tests/test_scrapers.py:57]: @DavidLemayian back to this conversation, you see for the dummy data, it's included in the repo where as the data directory which is a substitute for the aws bucket, is ignored. That's why I strongly suggest having the dummy data not included in the data dir. Another work around would be to have the tests read from the archived data but as we know those are a lot of records that will delay the testing.

Comment (DavidLemayian) [reply healthtools/tests/test_scrapers.py:57]: @Gathondu Yip. Sounds good. If the `TEST_DIR` doesn't change, should we therefore not have it config but instead just a constant in the test class?

Comment (Gathondu) [reply healthtools/tests/test_scrapers.py:57]: @DavidLemayian makes sense. Let me refactor the code.

**Pull request closed without merging**